revolv/templates/project/project.html

```diff
@@ -77,12 +80,32 @@
         <!-- These two are in here for testing and viewing purposes.  -->
         <!-- <div class="category-holder"><p class="category-label">ARTS</p></div>
         <div class="category-holder"><p class="category-label">HEALTH</p></div> -->
-    </div>
-    {% if project.video_url %}
-        <iframe width="100%" height="500" type="text/html" class="project_video youtube-player" frameborder="0" allowfullscreen></iframe>
-    {% endif %}
+        </div>
+        {% if project.video_url %}
+            <iframe width="100%" height="500" type="text/html" class="project_video youtube-player" frameborder="0" allowfullscreen></iframe>
+        {% endif %}
 
-    <div class="about-text"><p>{{ project.org_about }}</p>
+        <div class="about-text"><p>{{ project.org_about }}</p>
+        </div>
+    </div>
+    <div class="large-4 columns rewards-container">
+        <div class="rewards-text"><p class="rewards-text-inside">REWARDS</p></div>
+        <div class="gray-bar"></div>
+        {% for donation_level in project.donation_levels %}
+            <div class="donation-level-box">
+                <div class="donation-level-header">${{ donation_level.amount }}</div>
+                <div class="donation-level-text">{{ donation_level.description }}</div>
+            </div>
+        {% endfor %}
+            <!-- UNCOMMENT THESE BOXES TO SEE SAMPLE DONATION LEVELS  -->
```

Comment:
This is a lot of comments, how do you feel about that? Seems a little unprofessional now that I think about it?